plasmapy/particles/ionization_state.py

```diff
@@ -710,6 +714,14 @@ def base_particle(self) -> str:
         """The symbol of the element or isotope."""
         return self.isotope if self.isotope else self.element
 
+    @property
+    def as_particle_list(self) -> ParticleList:
+        """
+        A `~plasmapy.particles.particle_collections.ParticleList` of
+        the ionic levels.  The index corresponds to the charge number.
+        """
+        return self._particle_list
```

Comment:
This is why I was harping on `functools.cached_property` a while back; since you're not using `self._particle_list` anywhere else, using that decorator here and moving the definition of `self._particle_list` here would get us more "spatially concentrated" code, if you catch my cross-field drift.